backend/s3/s3.go

```diff
@@ -3605,6 +3663,21 @@ func (o *Object) Size() int64 {
 	return o.bytes
 }
 
+// ACL returns an object ACL and a bucket owner object
+func (o *Object) ACL(ctx context.Context) (string, string, error) {
```

Comment:
I don't think this function is called from `backend/s3` and we don't want to add a new public method to Object so I think it can be deleted.